### ISSUE TITLE
LoadingBar and Launching text

### ISSUE THREAD

**Issue by od040491mai** [djyde/WebShell#78]
Hi!Loading Animation, LoadingBar, Launching text Doesn't work with any website regardless of the change in settings.swift. In your example too.
first runs white blank with title "WebShell" for 1-2 seconds and only after website
![screenshot](https://user-images.githubusercontent.com/13697944/32796356-2f2cc668-c977-11e7-9b21-bcb3a788cb29.png)
website.

**Comment by 0xWDG** [comment]
I don’t understand what you mean.
But I will take a look somewhere next week.

The loading animation doesn’t work?
Or only on the first page?

**Comment by od040491mai** [comment]
In your demo StackEdit The launch of the application looks like this
![2017-11-14 20 20 08](https://user-images.githubusercontent.com/13697944/32797137-7760b302-c979-11e7-8bd7-9e0c836f2c43.png)

but in the Webshell-master it's only white blank, as in the first screenshot

**Comment by 0xWDG** [comment]
Ok, Will look somewhere next week

**Comment by 0xWDG** [comment]
Sorry for the delay, there was also another issue #80 which caused this weird behavior of WebShell.
